extensions/amp-base-carousel/1.0/amp-base-carousel.js

```diff
@@ -101,6 +101,13 @@ AmpBaseCarousel['children'] = {
     selector: '[slot="next-arrow"]',
     single: true,
   },
+  'pagination': {
+    name: 'pagination',
+    selector: 'amp-inline-gallery-pagination[inset]',
+    // Note: All must be selected so they do not count as slides,
+    // however only one is visible since they are positioned atop one another.
```

Comment:
Alternatively we can have the following config: 
```
'pagination': {
    name: 'pagination',
    selector: 'amp-inline-gallery-pagination[inset]',
    single: true, // Render the first instance
}
'paginationExtra': {
    // Discarded elements
    name: 'paginationExtra',
    selector: 'amp-inline-gallery-pagination[inset]',
    single: false,
}
```

Mostly we just don't want these elements to count as slides, so either of these are okay. Neither of these make an accessibility difference since all pagination elements are hidden from the tree (otherwise I'd advocate for the less verbose approach).